src/atom-environment.coffee

```diff
@@ -885,8 +885,6 @@ class AtomEnvironment extends Model
         else
           @project.addPath(pathToOpen)
 
-      @applicationDelegate.addRecentDocument(pathToOpen)
-
```

Comment:
@maxbrunsfeld I added the Recent Documents feature in https://github.com/atom/atom/pull/9609 because I wanted a way to quickly jump to "subjects" I recently worked on, where the subject can be either a project (folder) or a single file, but always an item I explicitly opened in a new Atom window.

![screen shot 2016-02-03 at 21 58 04](https://cloud.githubusercontent.com/assets/159434/12796857/4384be24-cac1-11e5-925c-9dda89d127e7.png)

If all files that are ever opened in a tab are added to Recent Documents, it loses all of its value for me, since the menu will always show 10 random files from the last project I worked with.

I don't see how that would actually be useful for anyone.